src/objects/core/models.py

```diff
@@ -40,6 +40,17 @@ def url(self):
         # zds_client.get_operation_url() can be used here but it increases HTTP overhead
         return f"{self.service.api_root}objecttypes/{self.uuid}"
 
+    @property
+    def version(self):
```

Comment:
It's not a version, it's rather the last available version. But the thing is that we specify the objecttype version in the object data, Objecttype model has only a link to objecttype itself.
@joeribekker Do we really want in the list admin view to perform a lot of HTTP requests to get the last version? 

Even if you want to go this was, I think it's more useful to show the list of all versions available, not the last one. 
And if you want to see them in the list view, it's better to perform one list request to Objecttypes API instead of a lot of individual detail requests

Reply:
I think I specced this ticket too easily. And we dont want to do many http calls here. I will respec the ticket.